src/textual/dom.py

```diff
@@ -561,6 +564,12 @@ def ancestors(self) -> list[DOMNode]:
             node = node._parent
         return cast("list[DOMNode]", nodes)
 
+    @property
+    def ancestors(self) -> list[DOMNode]:
+        """list[DOMNode]: A list of ancestor nodes Nodes by tracing ancestors all the way back to App."""
+        nodes = self.ancestors_with_self
+        return nodes[1:] if nodes else nodes
```

Comment:
I think we can guarantee that nodes has at least one node.

Reply:
Good point. :-D